indexing-hadoop/src/main/java/io/druid/indexer/JobHelper.java

```diff
@@ -93,6 +93,33 @@ public static Path distributedClassPath(Path base)
   }
 
   /**
+   * Dose authenticate against a secured hadoop cluster
+   * In case of any bug fix make sure to fix the code at HdfsStorageAuthentication#authenticate as well.
+   * @param config containing the principal name and keytab path.
+   */
+  public static void authenticate(HadoopDruidIndexerConfig config)
```

Comment:
is there any way to avoid code duplication ? 
maybe if instead of HadoopDruidIndexerConfig we only take principal and keytab ?